src/provider.toit

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) 2025 Toit contributors.
+// Use of this source code is governed by an MIT-style license that can be found
+// in the LICENSE file.
+
+import gpio
+import i2c
+import sensors.providers
+
+import .driver as bme280
+
+NAME ::= "toit.io/bme280"
```

Comment:
You're alternating between using toit.io and toitware as the prefix here. Do you have a plan for which one you want to use going forward?

Reply:
For the URI I try to use something that goes to the repository. For now this means "toitware".
Otherwise, I try to avoid toitware.